### DISCUSSION TITLE
Configure Capistrano for Vite Ruby

### DISCUSSION THREAD

**Discussion by brendon** [ElMassimo/vite_ruby#235] [Ideas]
**Is your feature request related to a problem? Please describe.**
If `public/vite` isn't added to `linked_dirs` in Capistrano then the second deploy will not compile any assets and Capistrano will blow the existing assets away resulting in no assets.

**Describe the solution you'd like**
Rails adds `public/assets` to the `linked_dirs` array in the following method: https://github.com/capistrano/rails/blob/888418b7ccb2b8cdb136cf7fa471044de855c4b0/lib/capistrano/tasks/assets.rake#L122-L130

We could probably do the same with `vite_ruby`. What do you think?

**Comment by ElMassimo** [comment]
Hi Brendon!

How to integrate with Capistrano is [documented in the _Deployment_ section](https://vite-ruby.netlify.app/guide/deployment.html#using-capistrano).

---

> Rails adds public/assets to the linked_dirs array in the following method

It's `capistrano-rails`, not Rails itself.

Configuring Capistrano is out of scope for `vite_ruby`, but having a `capistrano-vite` gem could be nice.

@mattbrictson might have some thoughts around what would be the best way to configure Capistrano + Vite Ruby.

**Comment by brendon** [comment]
Thanks @ElMassimo, I totally missed that! Or I think when I read it I was only starting out the conversion to Vite so I didn't take in the details. In my case I'm still using Sprockets for some css so I don't think I should change the `:assets_prefix` config. Adding `public/vite` to :`linked_dirs` seemed to do the trick in this case :)
